build_runner/lib/src/package_graph/apply_builders.dart

```diff
@@ -81,9 +87,27 @@ BuilderApplication apply(String builderKey,
       appliesBuilders: appliesBuilders,
     );
 
-/// A description of which packages need a given [Builder] applied.
+/// Same as [apply] except it takes [PostProcessBuilderFactory]s and does not
+/// provide options for `isOptional` or `hideOutput` because they aren't
+/// configurable for these types of builders. They are never optional and always
+/// hidden.
+BuilderApplication applyPostProcess(String builderKey,
+        List<PostProcessBuilderFactory> builderFactories, PackageFilter filter,
```

Comment:
`PostProcessBuilder` in `build_config` doesn't have an `auto_apply` - does it need `filter` here? Or can we start with it automatically being `toNoneByDefault()` and add an optional arg later if we need it?

It also doesn't have `applies_builders`, does it need it here?

https://github.com/dart-lang/build/blob/a98f4b12e0344ee3b8e8fb079d073e4689221036/build_config/lib/src/parse.dart#L43